site/sass/components/_item-grid.scss

```
@@ -14,25 +14,50 @@
   position: relative;
   z-index: 10;
 
+  &__title,
+  &__subtitle {
+    text-align: center;
+  }
+
   &__title {
     margin: 0 auto;
     max-width: pxRem(640px);
     padding: 0 calc(var(--shape-circle--size) / 6);
     position: relative;
-    text-align: center;
     z-index: 3;
   }
 
+  &__subtitle {
+    margin: pxRem(8px) auto 0;
+    max-width: pxRem(450px);
+  }
+
   &__content {
     display: grid;
-    grid-gap: pxRem(40px) pxRem(24px);
+    grid-gap: pxRem(20px) pxRem(20px);
```

```suggestion
    gap: 1.25rem
```

Two comments here:
* It turns out, we were wrong when we wrote this originally! While lots of browsers support `grid-gap`, the standardized property name is `gap`, so we should use that instead
* Because this is a shorthand value and we want them to be set the same, we can just specify a single value instead of the same one twice
* Updating to use REMs directly as we have a preference for doing so where possible